### PR TITLE
22468 - Launch Tiles

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:*
https://github.com/bcgov/entity/issues/22468

*Notes:*
- Requires the addition of feature flag: `enable-drs-lookup` to whichever LD space manages auth-web
- Adds env/env.example and vaults properties for document service url 
- *this may require the property to be added if the vault properties are not shared between projects

*Description of changes:*
- Creates common component launch tile for basic feature routing or external connections (ie an action or href) 
- Refactors Involuntary Dissolution tile to a configuration (and adds Document Record Service config)
- Unit testing for common component
- Linting / Spacing fixes

Screenshot 2024-10-31 at 8 26 31 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3134-8p6hibgx.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3134-8p6hibgx.web.app

**Comment by cameron-eyds** [comment]
Not sure about this, didn't like it used directly from prop, so tried computing it via the prop.
Either way, the url is provided and computed from local configuration / env variables, not sure what more we can do to sanitize, open to suggestions! 

Screenshot 2024-10-31 at 9 21 29 AM

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3134-8p6hibgx.web.app

**Comment by seeker25** [comment]
@cameron-eyds  can you put the ticket number in the title please? we have an automated release script that picks these up and tags them in zenhub

**Comment by seeker25** [comment]
> Not sure about this, didn't like it used directly from prop, so tried computing it via the prop. Either way, the url is provided and computed from local configuration / env variables, not sure what more we can do to sanitize, open to suggestions!
> 
> Screenshot 2024-10-31 at 9 21 29 AM

I think there is a sanitize library we're already using

**Comment by cameron-eyds** [comment]
> @cameron-eyds can you put the ticket number in the title please? we have an automated release script that picks these up and tags them in zenhub
> 
> https://github.com/bcgov/sbc-pay/blob/main/releases/release_to_zenhub_auth.py

Absolutely, thank you!

**Review by seeker25 — APPROVED** [review]
Let me know if I should create the flag in LD, or if you'll handle this one

also need secrets in 1pass

File: auth-web/devops/vaults.env [hunk 1]
```diff
@@ -15,6 +15,7 @@ VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
 VUE_APP_AUTH_WEB_URL="op://web-url/$APP_ENV/auth-web/AUTH_WEB_URL"
 VUE_APP_DASHBOARD_URL="op://web-url/$APP_ENV/business/DASHBOARD_URL"
+VUE_APP_DOCUMENTS_UI_URL="op://web-url/$APP_ENV/documents-ui/DOCUMENTS_UI_URL"
```
Comment (seeker25) [inline auth-web/devops/vaults.env:18]: need these created in 1pass

**Comment by cameron-eyds** [comment]
> Let me know if I should create the flag in LD, or if you'll handle this one
> 
> also need secrets in 1pass

@seeker25 
Yes please, thank you!
I don't have LD admin access, that would awesome. 
If you could set to `true` in DEV and `false` for TEST/PROD

File: auth-web/devops/vaults.env (hunk 1 above)
Comment (cameron-eyds) [reply auth-web/devops/vaults.env:18]: Copy that, wasn't sure if there was a global 1pass or per product etc. 
If there is one for auth-web, will definitely need it yessir. 

Also why i tagged Patrick Wei as well, a piece i don't have access to.

**Comment by seeker25** [comment]
Tomorrow we'll get this sorted

**Comment by cameron-eyds** [comment]
> Tomorrow we'll get this sorted

No rush whatsoever, whenever works!

**Pull request merged**